src/backend/index.js

```diff
@@ -0,0 +1,58 @@
+const express = require("express");
+const app = express();
+
+
+
+
+
+app.get("/", (request, response) => {
+  response.send(`
+    <body>
+        <h1>Index page </h1>
+        <p>Im inside index</p>
+    </body>
+  `);
+});
+
+
+// for meal page
+
+const mealRouter = require("./routes/meal.js");

```

Comment:
`mealRoute` would be a better name because "router" has a different meaning.

Reply:
It is not just this `const`, but the same applies to the other ones defined below.